assets/prototype/domain/shared/dateRangeInput/DateRangeInput.js

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import isFunction from 'lodash/isFunction';
+import moment from 'moment';
+import {
+	DateRangeInput as DefaultDateRangeInput,
+	TimePrecision
+} from '@blueprintjs/datetime';
+import { FORMATS } from './formatSelect';
+import { MomentDateRange } from './momentDate';
+
+import './style.css';
```

Comment:
@tn3rb did you tried to override some `blueprint`'s styles?
I tried here, but I am missing something.